### DISCUSSION TITLE
z Index for MVTLayer

### DISCUSSION THREAD

**Discussion by invisibru** [visgl/deck.gl#8702] [Q&A]
Hello. We use the deck.gl in integration with Openlayers. Drawing the MVTLayer on the openlayers map.
The openlayers parameter of the z Index layer does not work for MVTLayer. And the MVTLayer always lies above the rest of the openlayers layers.
Please tell me how to set the z Index for MVTLayer

**Comment by pReya** [comment]
Same Problem here. The MVTLayer is always on top of everything else (e.g. it's also covering up the AttributionControl).
